ethcore/types/src/transaction/transaction.rs

```diff
@@ -245,9 +416,12 @@ impl Transaction {
         SignedTransaction {
             transaction: UnverifiedTransaction {
                 unsigned: self,
-                r: U256::one(),
-                s: U256::one(),
-                v: 0,
+                chain_id: None,
+                signature: SignatureComponents {
+                    r: U256::one(),
+                    s: U256::one(),
+                    v: 4,
```

Comment:
Why this `4` here?

Reply:
It should represent an invalid signature. when `chain_id` and `standard_v` were merged this value was `0`. Why I use exactly `4` is because of [this ](https://github.com/openethereum/openethereum/blob/bf5830f766bf89861b92824de08f92a00de79d44/ethcore/types/src/transaction/transaction.rs#L102) code that marks `standard_v` invalid as `4`.

Reply:
Because of this `4` i needed to change transaction hashes of few tests.